src/common/actions/snaps.js

```diff
@@ -5,7 +5,7 @@ import { conf } from '../helpers/config';
 
 const BASE_URL = conf.get('BASE_URL');
 
-export const FETCH_SNAPS = 'FETCH_REPOSITORIES';
+export const FETCH_SNAPS = 'FETCH_SNAP_REPOSITORIES';
```

Comment:
Since I'm not changing the name of the const only it's value this should have no effect other that to help people who grep and potentially avoid bugs in reducers.